search.yaml

```
@@ -0,0 +1,433 @@
+        description: Filter by program germplasmId.
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: string
+      - name: treatmentId
+        in: query
+        description: Filter by treatment.
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: string
+      - name: page
+        in: query
+        description: Which result page is requested. The page indexing starts at 0
+          (the first page is 'page'= 0). Default is `0`.
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: integer
+        example: "0"
+      - name: pageSize
+        in: query
+        description: The size of the pages to be returned. Default is `1000`.
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: integer
+        example: 1000
+      - name: Authorization
+        in: header
+        description: "HTTP HEADER - Token used for Authorization \n\n<strong>Bearer\
+          \ {token_string} </strong>"
+        required: false
+        style: simple
+        explode: false
+        schema:
+          pattern: ^Bearer .*$
+          type: string
+        example: Bearer XXXX
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/searchResults'
+              example:
+                metadata:
+                  datafiles: []
+                  pagination:
+                    currentPage: 0
+                    pageSize: 2
+                    totalCount: 2
+                    totalPages: 1
+                  status: []
+                result:
+                  data:
+                    leadPersonDbId: person1
+                    leadPersonName: Name Nameson
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                type: string
+              example: |-
+                ERROR - 2018-10-08T20:15:11Z - Malformed JSON Request Object
+                ERROR - 2018-10-08T20:15:11Z - Invalid query parameter
+                ERROR - 2018-10-08T20:15:11Z - Required parameter is missing
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                type: string
+              example: ERROR - 2018-10-08T20:15:11Z - Missing or expired authorization
+                token
+        403:
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                type: string
+              example: ERROR - 2018-10-08T20:15:11Z - User does not have permission
+                to perform this action
+  /seasons:
+    get:
+      tags:
+      - Seasons
+      summary: Get seasons with filters applied
+      description: Season method.
```

“Season method” seems ambiguous (b/c scientists use ‘method’ to refer to how things are measured in the field). I realize this may be hidden in the API specs, but something like 
```suggestion
      description: API Method for Season
```

Might be more specific

---

changed